### PR TITLE
fix: migration

### PR DESCRIPTION
**Pull request by Kpoke**
Initial draft for the migration script, it is to support the creation of treetracker.capture as well

### CODE REVIEW

File: v2Tov1Migrations/migrate_regions.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+require('dotenv').config();
+const ProgressBar = require('progress');
+const { Writable } = require('stream');
+
+const ws = Writable({ objectMode: true });
+const { knex } = require('../database/knex');
+
+async function migrate() {
+  const base_query_string = `SELECT pr.* FROM public.region pr left join regions.region rr on pr.region_uuid = rr.id 
+  where rr.id is null and pr.type_id = 2`;
```
Comment (dadiorchen) [inline v2Tov1Migrations/migrate_regions.js]: @Kpoke can you explain this?

File: v2Tov1Migrations/helper/createCapture.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      .insert({ capture_id: captureToCreate.id, tag_id: uuid })
+      .into('treetracker.capture_tag');
+  }
+
+  // raw captures created before verify tool was moved to the microservices
+  if (rawCapture.status !== 'approved') {
+    await trx
+      .update('field_data.wallet_registration')
+      .set({ status: 'approved' })
+      .where({ id: rawCapture.id });
```
Comment (dadiorchen) [inline v2Tov1Migrations/helper/createCapture.js]: @Kpoke why we also need status:approved for the wallet_registration table?

File: v2Tov1Migrations/migrate_regions.js (hunk 1 above)
Comment (Kpoke) [reply v2Tov1Migrations/migrate_regions.js]: The regions table is so large about 6 million entries and I am not sure all of them are being used, so my idea for this is to migrate this based on the types that are needed, like freetown, then grids etc.

File: v2Tov1Migrations/helper/createCapture.js (hunk 2 above)
Comment (Kpoke) [reply v2Tov1Migrations/helper/createCapture.js]: This is a typo, that should be field_data.raw_capture, that would be for existing raw captures in production, the verify tool in production does not currently affect them

**Pull request merged**